config/environments/production.rb

```diff
@@ -1,7 +1,8 @@
 Rails.application.configure do
   config.action_mailer.default_url_options = { host: "http://wauwau.club" }
   # Settings specified here will take precedence over those in config/application.rb.
-
+  config.action_mailer.delivery_method = :smtp
+  config.action_mailer.default_url_options = { host: "wauwau-app.herokuapp.com" }
```

Comment:
You don't need this line, the `config.action_mailer.default_url_options` is already defined on line 2.